2-mandatory/8-password-validator.js

```diff
@@ -23,50 +23,68 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
```

Comment:
Great job Haroon! in this exercise one of the conditions is that Must not be any previous password in the passwords array that i think you forget to consider it.

Reply:
Thank you 😄. I didn't forget it. the isNotRepeated() function does that 😃.